patches/kea-forms@3.0.1.patch

```diff
@@ -0,0 +1,37 @@
+diff --git a/lib/types.d.ts b/lib/types.d.ts
```

Comment:
CC @mariusandra

Reply:
Ah, cool, yeah an upstream fix would be even better. 🙂 

The issue here is that `kea` in 3.0.0-alpha accepted just two generic arguments for `SelectorDefinition`, but in the 3.x.x released version requires three. Looks like `kea-forms` assumes the alpha version's types.

Reply:
Thanks for finding this! I released version 3.0.2 of kea-forms with the fix.